package.json

```diff
@@ -270,7 +270,8 @@
   },
   "workspaces": {
     "packages": [
-      "packages/*"
+      "packages/*",
+      "plugins-bundled/internal/*"
```

Comment:
just making stuff up!  but this must do something!

Reply:
why do we need this? bundled plugins should not contain npm packages that we import from other packages / core

Reply:
@dprokop ?  is this necessary?  would not be used for npm/webpack, only vscode